### ISSUE TITLE
Character counter in organization description

### ISSUE THREAD

**Issue by Dissfall** [diglabby/mapa#226]
It seems to me that it is much more convenient to write a description if you know how many characters you have left to write at the moment. I propose instead of the text under the description input field to put a counter that displays the number of remaining characters (or even a counter in the form of a filling circle as in Twitter).

**Comment by rizomaa** [comment]
@Dissfall yep, it sounds reasonable. If you want this feature will be implemented make PR with your code edits) Any way thanks for the idea.

**Comment by mvit22** [comment]
Hello, I made a restriction on the characters in the description.
![image](https://user-images.githubusercontent.com/43513348/82727701-cb995b80-9cf4-11ea-8ad0-392c5224249c.png)

**Comment by rizomaa** [comment]
@VitalyMikulich looks good.
Please add here the Pull Request for this changes.

**Comment by VitalyMikulich** [comment]
https://github.com/diglabby/mapa/pull/348

**Comment by mvit22** [comment]
Link to PR https://github.com/diglabby/mapa/pull/348

**Comment by rizomaa** [comment]
@VitalyMikulich пракаментаваў ПР
